tests/test-circular-layout-computation.js

```diff
@@ -72,40 +74,40 @@ require(['jquery', 'BPTree', 'Empress'], function($, BPTree, Empress) {
                     {},
                     null
                 );
-                this.empress._drawer = new Object();
+                this.empress._drawer = {};
                 this.empress._drawer.VERTEX_SIZE = 5;
             },
 
-            teardown: function() {
+            teardown: function () {
                 this.empress = null;
-            }
+            },
         });
 
-        test('Test Circular Layout Arc Computation', function() {
+        test("Test Circular Layout Arc Computation", function () {
             var coords = this.empress.getCoords();
 
             // NOTE: all node numbers are in reference to the postorder position
             //      starting at 1.
             // check if line for node 1 is correct (tip)
-            var node = 1
-            equal(coords[(node-1)*10], -2); // start x position
-            equal(coords[(node-1)*10 + 1], 2); // start y position
-            equal(coords[(node-1)*10 + 5], -2); // end x position
-            equal(coords[(node-1)*10 + 6], 0); // end y position
+            var node = 1;
+            equal(coords[(node - 1) * 10], -2); // start x position
+            equal(coords[(node - 1) * 10 + 1], 2); // start y position
+            equal(coords[(node - 1) * 10 + 5], -2); // end x position
+            equal(coords[(node - 1) * 10 + 6], 0); // end y position
 
             // check if line for node 3 is correct (internal)
             node = 3;
-            equal(coords[(node-1)*10], 0); // start x position
-            equal(coords[(node-1)*10 + 1], 1); // start y position
-            equal(coords[(node-1)*10 + 5], 0); // end x position
-            equal(coords[(node-1)*10 + 6], -1); // end y position
+            equal(coords[(node - 1) * 10], 0); // start x position
+            equal(coords[(node - 1) * 10 + 1], 1); // start y position
+            equal(coords[(node - 1) * 10 + 5], 0); // end x position
+            equal(coords[(node - 1) * 10 + 6], -1); // end y position
 
             // For the arc for node 3 start at (2,0) and ends at (-2, 0)
             // check if arc for node 3 is correct
             ok(Math.abs(coords[30] - 2) < 1.0e-15); // start x arc position
             ok(Math.abs(coords[31 - 0]) < 1.0e-15); //start y arc position
-            ok(Math.abs(coords[175] - -2) < 1.0e-15) // end x arc position
-            ok(Math.abs(coords[176] - 0 < 1.0e-15)) // end y arc position
+            ok(Math.abs(coords[175] + 2) < 1.0e-15); // end x arc position
```

Comment:
```suggestion
            ok(Math.abs(coords[175] - (-2)) < 1.0e-15); // end x arc position
```
This is not a big dead so if the linter complains do worry about it.